transaction-overview.md

```diff
@@ -62,42 +72,104 @@ Syntax:
 ROLLBACK;
 ```
 
-You can use this statement to roll back the current transaction and cancels all updates between `[BEGIN | START TRANSACTION]` and `ROLLBACK`.
+Transactions are also automatically rolled back if the client connection is aborted or closed.
 
 ## Autocommit
 
-Syntax:
+As part of MySQL compatibility, TiDB supports the ability to _autocommit_ a statement immediately following its execution.
```

Comment:
Perhaps it's weird to say TiDB is part of xx compatibility? Generally, we would say TiDB is a product/database/....

Reply:
It's a polite way of saying "this is not TiDB's fault, it has to act in this silly way to be MySQL compatible". It could also say "As required for MySQL compatibility, TiDB will by default _autocommit_ statements immediately following their execution."

Reply:
I prefer the new version.